torchao/sparsity/README.md

```diff
@@ -0,0 +1,664 @@
+# torchao sparsity
+
+Sparsity is the technique of removing parameters from a neural network in order to reduce its memory overhead or latency. By carefully choosing the elements that are removed, one can achieve significant reduction in memory overhead and latency, while paying a reasonably low or no price in terms of model quality (accuracy / f1).
```

Comment:
I'd call pruning the "technique of removing parameters from a neural network in order to reduce its memory overhead or latency"